noir_stdlib/src/hash.nr

```diff
@@ -6,12 +6,12 @@ pub fn sha256<N>(_input : [u8; N]) -> [u8; 32] {}
 #[foreign(blake2s)]
 pub fn blake2s<N>(_input : [u8; N]) -> [u8; 32] {}
 
-pub fn pedersen<N>(input : [Field; N]) -> [Field; 2] {
-    pedersen_with_separator(input, 0)
+pub fn pedersen_commitment<N>(input : [Field; N]) -> [Field; 2] {
+    pedersen_commitment_with_separator(input, 0)
 }
```

Comment:
The namespace for pedersen_commitment now looks somewhat odd `std::hash::pedersen_commitment` -- A possibly better one would be `crate::pedersen::commitment` alongside ``crate::pedersen::hash`. The downside being that pedersen hash is not in the hash namespace